README.md

```diff
@@ -20,13 +20,34 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [TensorFlow 1.0 or 1.1](https://www.tensorflow.org/install/)
 * [SciPy](https://scipy.org/install.html)
 * [PyXDG](https://pypi.python.org/pypi/pyxdg)
-* [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 * [pandas](https://pypi.python.org/pypi/pandas)
+* [DeepSpeech native client libraries](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/)
 
 
 ## Install
 
+### Installing pre-built DeepSpeech Python bindings
+
+Pre-built binaries can be found on TaskCluster. You'll need to download `native_client.xz` and the appropriate Python wheel package.
+
+[native_client.tar.xz (Linux / amd64)](https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.cpu/artifacts/public/native_client.tar.xz)
+[deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl (Linux / amd64)](https://index.taskcluster.net/v1/task/project.deepspeech.deepspeech.native_client.master.cpu/artifacts/public/deepspeech-0.0.1-cp27-cp27mu-linux_x86_64.whl)
+[Other configurations](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master)
+
+First, the library files contained in `native_client.xz` need to be installed within the system library path.
```

Comment:
nit: The file is actually called `native_client.tar.xz`.

I think this is a bit too confident about people's Linux skills. I'd mention /usr/lib or LD_LIBRARY_PATH explicitly here. Something like:

First, the library files contained in `native_client.tar.xz` need to be installed within the system library path. You have two options:

1. Extract the contents of `native_client.tar.xz` in `/usr/lib`.
2. Extract them anywhere, and then use `LD_LIBRARY_PATH` when running the code: `LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/path/to/where/you/extracted/the/files ./bin/run-ldc93s1.sh`